src/objects/Hierarchy_Settings__c.object

```diff
@@ -390,6 +390,16 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Validate_Program_Plan_for_Nested_PR__c</fullName>
+        <defaultValue>true</defaultValue>
```

Comment:
@xinnanlisf @lifewithryan  Did we decide that this should be disabled by default for all current customers, but enabled for new installs? If so, I think we need additional changes here. Either this should be defaulted to false and set to true on new installs, or set to true by default and set to false when upgrading from 1.77 to 1.78... thoughts? (Or maybe this works as expected?)

Reply:
If I remember it correctly, upgrading EDA won't populate the custom setting value for existing users.

Reply:
I thought that was only on existing fields; don't brand new fields come over with a value? Since its a checkbox, I thought we had an issue where it is neither true nor false until it's toggled. (I could be wrong though).
I will ask since I clearly can't remeber :)

Reply:
I can't quite remember either - I'm leaning towards @xinnanlisf being correct. We may need to wait for the beta to know for certain. Either way, it should definitely be part of the test plan.

Reply:
Just confirmed in my own packaging test org. Checkbox will be unchecked when upgrading even the default value is "checked".

Reply:
@xinnanlisf Perfect!!!